src/main/java/dev/nolij/zume/FabricZumeBootstrapper.java

```diff
@@ -12,10 +12,10 @@
 public class FabricZumeBootstrapper implements ClientModInitializer, PreLaunchEntrypoint {
 	
 	private static final String MISSING_DEPENDENCY_MESSAGE = """
-		Failed to detect which variant of Zume to load! Ensure all dependencies are installed:
-		    Fabric (14.4+): Fabric API (fabric-key-binding-api-v1)
-		    Legacy Fabric (7.10-12.2): Legacy Fabric API (legacy-fabric-keybinding-api-v1-common)
-		    Babric (b7.3): Station API (station-keybindings-v0)""";
+	                                                         Failed to detect which variant of Zume to load! Ensure all dependencies are installed:
```

Comment:
This feels like too much indentation. Maybe move the `"""` to the next line to avoid it?